server/server.go

```diff
@@ -48,7 +56,9 @@ func ListenAndServe(ctx context.Context, httpsPort, httpPort int, handler http.H
 	}
 
 	logger := logrus.StandardLogger()
-	errorLog := log.New(logger.WriterLevel(logrus.DebugLevel), "", log.LstdFlags)
+	logger.SetLevel(logrus.DebugLevel)
```

Comment:
I'm not sure that this solves the issue. From the issue, my understanding of the problem was that we were using logrus at debug level. Since logrus [defaults to info](https://pkg.go.dev/github.com/sirupsen/logrus#Logger), this effectively suppressed any output in default circumstances.

This PR does not seem to fix that - it's still at debug level and cannot be overriden by an argument in the options. Did I misunderstand the original issue?

Reply:
First change: log the panics from the server. I don't think they were even getting to the `dynamicListener` client.

Reply:
Second change: honor the same env vars that `webhook` does to put the logger in debug mode.